docs/rules/no-extra-boolean-cast.md

```diff
@@ -68,3 +68,53 @@ function foo() {
 
 var foo = bar ? !!baz : !!bat;
 ```
+
+## Options
+
+This rule has an object option:
+
+* `"enforceForLogicalOperands"` when set to `true`, checks whether the extra boolean is contained within a logical expression. Default is `false`, meaning that this rule by default does not warn about extra booleans inside logical expression.
+
+### enforceForLogicalOperands
+
+Examples of **incorrect** code for this rule with `"enforceForLogicalOperands"` option set to `true`:
+
+```js
+/*eslint no-extra-boolean-cast: ["error", {"enforceForLogicalOperands": true}]*/
```

Comment:
It would be nice to also have an example with an operand of `&&` (at the moment `&&` appears only in the example with nested expressions).